tests/src/integration/zipaction/manifest.yml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: zipaction
+  version: 1.0
+  license: Apache-2.0
+  actions:
+    cat:
+      version: 1.0
+      location: actions/cat
```

Comment:
This is a good simple (in-line), single action test (better than previous one).  We should perhaps create another test that uses multiple (sub)directories.